source/excerptexport/views.py

```diff
@@ -59,8 +63,13 @@ def create_excerpt_export(request):
         bounding_geometry.excerpt = excerpt
         bounding_geometry.save()
 
-        viewContext = { 'use_existing': False, 'excerpt': excerpt, 'bounding_geometry': bounding_geometry }
+        viewContext = { 'excerpt': excerpt, 'bounding_geometry': bounding_geometry }
+        ExtractionOrder.objects.create(
+            excerpt = excerpt,
+            orderer = request.user
+        )
 
+    viewContext['use_existing'] = 'existingExcerptID' in vars() # TODO: The view should not have to know
```

Comment:
Maybe a better solution: set 'excerpt' OR 'existing_excerpt' and let the view decide by check what is defined?